include/uprotocol-cpp/uri/validator/UriValidator.h

```diff
@@ -19,21 +19,92 @@
  * under the License.
  * 
  * SPDX-FileType: SOURCE
- * SPDX-FileCopyrightText: 2023 General Motors GTO LLC
+ * SPDX-FileCopyrightText: 2024 General Motors GTO LLC
```

Comment:
Copyright doesn't lose original start date: Change to: 2023-2024

Reply:
Done